test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -102,3 +102,5 @@ fun <T> KMutableProperty<T?>.require() =
     getter.call() ?: throw FlankGeneralError(
         "Invalid value for [${setter.annotations.filterIsInstance<JsonProperty>().first().value}]: no argument value found"
     )
+
+fun getGACPathOrNull(): String? = System.getenv("GOOGLE_APPLICATION_CREDENTIALS")
```

Comment:
Basically, it does not return path, but value of env variable

Reply:
True, but I wanted to be explicit that path (as a string) is the expected value here